cli/search/search.go

```diff
@@ -90,3 +93,106 @@ func SearchVersions(cmdCtx *cmdcontext.CmdCtx, program string) error {
 	os.Stdout.Write([]byte("master\n"))
 	return err
 }
+
+func getDirFiles(path string) ([]fs.FileInfo, error) {
+	f, err := os.Open(path)
+	if err != nil {
+		fmt.Println(err)
+		return nil, err
+	}
+	files, err := f.Readdir(0)
+	if err != nil {
+		fmt.Println(err)
+		return nil, err
+	}
+	return files, err
+}
+
+// RunCommandAndGetOutputDir returns output of command.
+func RunCommandAndGetOutputDir(program string, dir string, args ...string) (string, error) {
```

Comment:
I find "Dir" at the end of this function a little confusing.

Reply:
changed